File: fibonacci/fibonacci.R
fibonacci <- function(n) {
    if (n == 0 || n == 1) 1
    else fibonacci(n - 1) + fibonacci(n - 2)
}

Main <- function() {
    n <- 42
    result <- fibonacci(n)
    cat(result, fill = TRUE)
}

if (!interactive()) {
    Main()
}

File: fibonacci/fibonacci.c
#include <stdio.h>

int fibonacci(int n) {
   if (n == 0 || n == 1) {
      return 1;
   } else {
      return fibonacci(n - 1) + fibonacci(n - 2);
   }
}

void main() {
   int n = 42;
   int result = fibonacci(n);
   printf("%d", result);
}
